vip-filesystem/class.vip-filesystem-loader.php

```diff
@@ -0,0 +1,121 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: hanifn
+ * Date: 31/10/18
+ * Time: 11:48 PM
+ */
+
+namespace Automattic\VIP\Filesystem;
+
+/**
+ * Register all actions and filters for the plugin.
+ *
+ * Maintain a list of all hooks that are registered throughout
+ * the plugin, and register them with the WordPress API. Call the
+ * run function to execute the list of actions and filters.
+ */
+class Vip_Filesystem_Loader {
```

Comment:
What's the benefit to having this Loader class? Feels a bit like an unnecessary abstraction (and maybe created as part of some boilerplate?) If there are some clear benefits, we can keep it; otherwise, probably just simpler to call `add_action`/`add_filter` directly in the main `Vip_Filesystem` class.